### PR TITLE
Added add cart item & updated cart items endpoints

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [X] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to generate documentation for the "Add cart items" endpoint in Dev. Portal and update the description/ parameters of the "Update cart items" endpoint. (JIRA EDU-8206).

### CODE REVIEW

File: VTEX - Checkout API.json [hunk 1]
```diff
@@ -2134,14 +2134,1757 @@
...
+                                            "description": "User type.",
+                                            "nullable": true
+                                        },
+                                        "ignoreProfileData": {
+                                            "type": "boolean",
+                                            "description": "Ignore customer profile data."
+                                        },
+                                        "value": {
+                                            "type": "integer",
+                                            "description": "Total value of the order without separating cents. For example, $24.99 is represented `2499`."
```
Comment (karenkrieger) [inline VTEX - Checkout API.json:2360]:
```suggestion
                                            "description": "Total value of the order without separating cents. For example, $24.99 is represented as `2499`."
```

File: VTEX - Checkout API.json [hunk 2]
```diff
@@ -2134,14 +2134,1757 @@
...
+                                                                                    "quantity": {
+                                                                                        "type": "integer",
+                                                                                        "description": "Quantity."
+                                                                                    }
+                                                                                }
+                                                                            }
+                                                                        },
+                                                                        "shippingEstimate": {
+                                                                            "type": "string",
+                                                                            "description": "Shipping estimate. For instance, Three business days will be represented `3bd`."
```
Comment (karenkrieger) [inline VTEX - Checkout API.json:2797]:
```suggestion
                                                                            "description": "Shipping estimate. For instance, \"three business days\" will be represented as `3bd`."
```

File: VTEX - Checkout API.json [hunk 3]
```diff
@@ -2134,14 +2134,1757 @@
...
+                                                                            "description": "Pickup point distance."
+                                                                        },
+                                                                        "polygonName": {
+                                                                            "type": "string",
+                                                                            "description": "Polygon name.",
+                                                                            "nullable": true
+                                                                        },
+                                                                        "transitTime": {
+                                                                            "type": "string",
+                                                                            "description": "Transit time. For instance, \"three business days\" is represented `3bd`."
```
Comment (karenkrieger) [inline VTEX - Checkout API.json:2917]:
```suggestion
                                                                            "description": "Transit time. For instance, \"three business days\" is represented as `3bd`."
```

File: VTEX - Checkout API.json [hunk 4]
```diff
@@ -2134,14 +2134,1757 @@
...
+                                            }
+                                        },
+                                        "subscriptionData": {
+                                            "type": "object",
+                                            "description": "Subscription information.",
+                                            "nullable": true
+                                        },
+                                        "itemsOrdination": {
+                                            "type": "object",
+                                            "description": "Object containing information about the ordering of items within the orderForm.",
```
Comment (karenkrieger) [inline VTEX - Checkout API.json:3459]:
```suggestion
                                            "description": "Object containing information about the ordering of items within the `orderForm`.",
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Checkout API.json [hunk 5]
```diff
@@ -2134,14 +2134,1757 @@
                 "deprecated": false
             }
         },
+        "/api/checkout/pub/orderForm/{orderFormId}/items/update": {
+            "post": {
+                "tags": [
+                    "Shopping cart"
+                ],
+                "summary": "Update cart items",
+                "description": "You can use this request to:\n\r\n\r1. Change the quantity of one or more items in a specific cart.\n\r2. 2. Remove an item from the cart (by sending the `quantity` value = `0` in the request body).\r\n\r\n**Important**: To remove all items from the cart at the same time, use the [Remove all items](https://developers.vtex.com/vtex-rest-api/reference/removeallitems) endpoint.\r\n\r\nThe [orderForm](https://developers.vtex.com/vtex-rest-api/reference/checkout-api-overview) is the data structure which represents a shopping cart and contains all information pertaining to it. Hence, the `orderFormId` is the identification code of a given cart.\n\r\n\r> This request has a time out of 45 seconds.",
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Checkout API.json:2143]: Fixed.

File: VTEX - Checkout API.json [hunk 6]
```diff
@@ -2134,14 +2134,1757 @@
...
+                        "style": "simple",
+                        "schema": {
+                            "type": "string",
+                            "default": "application/json"
+                        }
+                    },
+                    {
+                        "name": "orderFormId",
+                        "in": "path",
+                        "description": "ID of the orderForm corresponding to the cart whose items you want to update.",
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Checkout API.json:2171]: Fixed.

File: VTEX - Checkout API.json (hunk 1 above)
Comment (ricardoaerobr) [reply VTEX - Checkout API.json:2360]: Fixed.

File: VTEX - Checkout API.json (hunk 3 above)
Comment (ricardoaerobr) [reply VTEX - Checkout API.json:2917]: Fixed.

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**